packages/react-utilities/package.json

```diff
@@ -20,9 +20,10 @@
     "lint": "just-scripts lint",
     "test": "jest",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
-    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/react-utilities/src && yarn docs",
+    "build:local": "tsc -p ./tsconfig.lib.json --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output ./dist/packages/react-utilities/src && yarn docs",
     "storybook": "start-storybook",
```

Comment:
_Just curiosity_: Is it expected to have Storybook there?

Reply:
I missed this one, I cleaned the .storybook and tsconfig

Reply:
I guess it snuck in from the last migration too